matplotlibrc.template

```diff
@@ -279,6 +280,22 @@ backend      : %(backend)s
 #polaraxes.grid      : True    # display grid on polar axes
 #axes3d.grid         : True    # display grid on 3d axes
 
+### DATES
+# These control the default format strings used in AutoDateFormatter.
+# Any valid format datetime format string can be used (see the python
+# `datetime` for details).  For example using '%%x' will use the locale date representation
+# '%%X' will use the locale time representation and '%%c' will use the full locale datetime
+# representation.
+# These values map to the scales:
```

Comment:
Might want to include a note that doubling up on the percent signs is only needed for the rcfile. I am assuming that setting the rcParam directly does not need doubling up on the percent signs?